### PR TITLE
Fix Streamlit in docs

### PR DESCRIPTION
**Pull request by sfc-gh-astus**
### Pre-review checklist
   * [x] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [x] I've added or updated automated unit tests to verify correctness of my new code.
   * [ ] I've added or updated integration tests to verify correctness of my new code.
   * [ ] I've confirmed that my changes are working by executing CLI's commands manually.
   * [x] I've confirmed that my changes are up-to-date with the target branch.
   * [ ] I've described my changes in the release notes.
   * [x] I've described my changes in the section below.

### Changes description
Changed `Streamlit` to `Streamlit app`

### CODE REVIEW

**Review by sfc-gh-turbaszek — APPROVED** [review]

File: src/snowflake/cli/plugins/streamlit/commands.py [hunk 1]
```diff
@@ -21,26 +21,30 @@
...
+    help="Manages Streamlit app in Snowflake.",
 )
 log = logging.getLogger(__name__)
 
 
 StageNameOption: str = typer.Option(
     "streamlit",
     "--stage",
-    help="Stage name where Streamlit files will be uploaded.",
+    help="Stage name where Streamlit app files will be uploaded.",
```
Comment (sfc-gh-mheavin) [inline src/snowflake/cli/plugins/streamlit/commands.py:32]: Name of the stage where you want to upload Streamlit app files

File: src/snowflake/cli/plugins/streamlit/commands.py [hunk 2]
```diff
@@ -21,26 +21,30 @@
...
+)
+
+OpenOption = typer.Option(
+    False, "--open", help="Whether to open Streamlit app in a browser.", is_flag=True
```
Comment (sfc-gh-mheavin) [inline src/snowflake/cli/plugins/streamlit/commands.py:36]: to open a Streamlit app

File: src/snowflake/cli/plugins/streamlit/commands.py [hunk 3]
```diff
@@ -21,26 +21,30 @@
...
+add_init_command(app, project_type="Streamlit app", template="default_streamlit")
 
 
 @app.command("share", requires_connection=True)
 def streamlit_share(
-    name: str = typer.Argument(..., help="Name of streamlit to share."),
+    name: str = typer.Argument(..., help="Name of Streamlit app to share."),
     to_role: str = typer.Argument(
-        ..., help="Role that streamlit should be shared with."
+        ..., help="Role that Streamlit app should be shared with."
```
Comment (sfc-gh-mheavin) [inline src/snowflake/cli/plugins/streamlit/commands.py:47]: Role with which to share the Streamlit app.

**Review by sfc-gh-mheavin — COMMENTED** [review]
Left some comments.

I did not continue with the "test" files, as they don't seem to be user-visible.

In general, please make sure you use "the" or "a" before "Streamlit app"

**Review by sfc-gh-mheavin — APPROVED** [review]
Looks good. Thanks.

**Review by sfc-gh-turbaszek — APPROVED** [review]

**Pull request merged**